index.js

```diff
@@ -1,14 +1,96 @@
 const express = require('express');
 const helmet = require('helmet');
-
+const knex = require('knex');
+const dbConfig = require('./knexfile');
+const { zooId404 } = require('./helpers/error');
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
 
+const db = knex({ ...dbConfig.development });
+
 // endpoints here
 
+server.get('/api/zoos', (req, res) => {
+    db('zoos')
+        .then(zoos => {
+            res.status(200).json({ zoos });
+        })
+        .catch(err => {
+            res.status(500).json(err);
+        });
+});
+
+server.post('/api/zoos', (req, res) => {
+    db('zoos')
+        .insert(req.body)
+        .then(result => {
+            res.status(201).json({ result });
+        })
+        .catch(err => {
+            res.status(500).json(err);
+        });
+});
+
+server.get('/api/zoos/:id', (req, res) => {
+    const { id } = req.params;
+    db('zoos')
+        .where('id', 'like', id)
+        .then(result => {
+            if (result.length === 0) {
+                zooId404(res);
+            } else {
+                res.status(200).json(result);
+            }
+        })
+        .catch(err => {
+            res.status(500).json(err);
+        });
```

Comment:
It's probably be worth it to have a server error helper function:
```javascript
const servErr = res => err => {
  res.status(500).json(err);
};
```
and usage (in all of your `.catch`s):
```javascript
.catch(servErr(res));
```